src/brasil/gov/portlets/portlets/audio.py

```diff
@@ -84,7 +85,7 @@ def audio(self):
         portal_state = getMultiAdapter((self.context, self.request),
                                        name=u'plone_portal_state')
         portal = portal_state.portal()
-        if isinstance(audio_path, unicode):
+        if isinstance(audio_path, str):
             # restrictedTraverse accepts only strings
             audio_path = str(audio_path)
```

Comment:
so, if it's already an instance of `str()` what's the point on converting it to `str()`?